pkg/run/utils.go

```diff
@@ -0,0 +1,218 @@
+package run
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"github.com/fluxcd/pkg/apis/meta"
+	"github.com/fsnotify/fsnotify"
+	"github.com/minio/minio-go/v7"
```

Comment:
Double-checked: The minio library is APL2, not AGPL.

Reply:
Just only the client library is APL2. 
The server is AGPL: https://github.com/minio/minio/blob/master/LICENSE